static-site/content/featured-analyses.yaml

```diff
@@ -20,26 +25,21 @@
   img: h5n1_cattle_outbreak_tree.png
   url: /avian-flu/h5n1-cattle-outbreak/genome
 
-- name: Avian influenza
-  description: Avian influenza A/H5N1 evolution and spread
-  img: h5n1_map.png
-  url: /avian-flu/h5n1/ha/2y
-
-- name: Mpox
-  description: Evolution and spread of mpox virus
-  img: mpox_tree.png
-  url: /mpox/clade-IIb
-
-- name: Ebola in the DRC
-  description: Genomic epidemiology of the 2018-20 Ebola epidemic in DRC
-  img: ebola_drc_map.png
-  url: /community/inrb-drc/ebola-nord-kivu
-
 - name: Seasonal influenza
   description: Seasonal influenza evolution and antigenic drift
   img: seasonal_flu_freq.png
   url: /seasonal-flu/h3n2/ha/2y
 
+- name: Lassa
+  description: Lassa virus evolution and spread
+  img: lassa_map.png
+  url: /lassa/l
+
+- name: Yersinia pestis
+  description: Historical analysis of geographic spread of plague
+  img: yersinia_tree.png
+  url: community/ktmeaton/yersinia-pestis/maximum-likelihood/all?m=num_date
```

Comment:
This is showing up as "Maintained by the Nextstrain team". The quick fix is to ensure URL starts with `/`. I'll do this. I'll also look into loosening that requirement or at least adding better checks.

Reply:
https://github.com/nextstrain/nextstrain.org/pull/995

Reply:
Thanks for fix!